playback.json

```diff
@@ -123,6 +123,12 @@
           "isRecursive": true,
           "annotation": "Functions for supporting FFW [Fizzi, Unclepunch]"
         },
+        {
+          "type": "injectFolder",
+          "sourceFolder": "Online/Core/EXIFileLoad",
```

Comment:
@JordanLozinski 
@JLaferri might be a good idea to move this to commons, since it found its way to non online usages.

Reply:
Not sure if you're asking me to move that as part of this PR or just suggesting this as future work